transport_analysis/viscosity.py

```diff
@@ -0,0 +1,191 @@
+"""
+Viscosity --- :mod:`transport_analysis.analysis.viscosity`
+==========================================================
+
+This module offers a class for the lightweight computation of
+viscosity via the Einstein-Helfand method.
+
+"""
+from typing import TYPE_CHECKING
+
+from MDAnalysis.analysis.base import AnalysisBase
+from MDAnalysis.core.groups import UpdatingAtomGroup
+from MDAnalysis.exceptions import NoDataError
+import numpy as np
+
+if TYPE_CHECKING:
+    from MDAnalysis.core.universe import AtomGroup
+
+
+class ViscosityHelfand(AnalysisBase):
+    """
```

Comment:
What's our plan for benchmarking the viscosity analysis?

Reply:
@xhgchen you can benmark as part of your reproduction.